### PR TITLE
update examples

### PR DESCRIPTION
**Pull request by xgqfrms-gildata**
# update examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,16 +198,20 @@ Other Style Guides
...
     const atom = {
       value: 1,
-
       addValue: function (value) {
         return atom.value + value;
       },
     };
 
-    // good
+    // very good
```
Comment (ljharb) [inline README.md]: this is actually worse than the following example, which uses concise method syntax.

File: README.md [hunk 2]
```diff
@@ -254,8 +258,8 @@ Other Style Guides
 
     // good
     const obj = {
+      anakinSkywalker,// keep up with declare order
```
Comment (ljharb) [inline README.md]: i don't think we need the comment, but the re-sorting is cute

File: README.md [hunk 3]
```diff
@@ -313,15 +317,15 @@ Other Style Guides
...
+    // not too bad
     const original = { a: 1, b: 2 };
     const copy = Object.assign({}, original, { c: 3 }); // copy => { a: 1, b: 2, c: 3 }
 
     // good
     const original = { a: 1, b: 2 };
     const copy = { ...original, c: 3 }; // copy => { a: 1, b: 2, c: 3 }
 
-    const { a, ...noA } = copy; // noA => { b: 2, c: 3 }
+    const { a, ...restA } = copy; // rest => { b: 2, c: 3 }
```
Comment (ljharb) [inline README.md]: `...restA` means that the resulting variable will be `restA`, not `rest`.

File: README.md [hunk 4]
```diff
@@ -505,6 +509,9 @@ Other Style Guides
     function getFullName({ firstName, lastName }) {
       return `${firstName} ${lastName}`;
     }
+    // perfect
```
Comment (ljharb) [inline README.md]: arrow functions are not better than actual named functions; please remove this.

File: README.md [hunk 5]
```diff
@@ -551,16 +561,16 @@ Other Style Guides
 ## Strings
 
   <a name="strings--quotes"></a><a name="6.1"></a>
-  - [6.1](#strings--quotes) Use single quotes `''` for strings. eslint: [`quotes`](https://eslint.org/docs/rules/quotes.html) jscs: [`validateQuoteMarks`](http://jscs.info/rule/validateQuoteMarks)
+  - [6.1](#strings--quotes) Use double quotes `""` for strings. eslint: [`quotes`](https://eslint.org/docs/rules/quotes.html) jscs: [`validateQuoteMarks`](http://jscs.info/rule/validateQuoteMarks)
```
Comment (ljharb) [inline README.md]: Please revert this section.

File: README.md [hunk 6]
```diff
@@ -581,8 +591,12 @@ Other Style Guides
       'of Batman. When you stop to think about how Batman had anything to do ' +
       'with this, you would get nowhere fast.';
 
-    // good
+    // not too good
```
Comment (ljharb) [inline README.md]: also please revert this. template literals are not permitted without interpolations.

File: README.md (hunk 2 above)
Comment (eddjoke) [reply README.md]: agree.

File: README.md (hunk 1 above)
Comment (dongsik2026) [reply README.md]: that is a good coding convention.

**Pull request closed without merging**